rosbag2_cpp/src/rosbag2_cpp/cache/cache_consumer.cpp

```diff
@@ -23,7 +23,7 @@ namespace cache
 {
 
 CacheConsumer::CacheConsumer(
-  std::shared_ptr<MessageCache> message_cache,
+  std::shared_ptr<rosbag2_cpp::cache_interfaces::BaseMessageCacheInterface> message_cache,
```

Comment:
Perhaps it could be shortened as:
```suggestion
  std::shared_ptr<rMessageCacheInterface> message_cache,
```